src/test/java/org/codewars/utils/ReaderTest.java

```diff
@@ -1,14 +1,23 @@
 package org.codewars.utils;
 
 import org.testng.Assert;
+
+import org.testng.Reporter;
+
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 import java.io.*;
+import java.math.BigInteger;
+import java.util.NoSuchElementException;
+
+import static org.testng.Assert.*;
```

Comment:
re-import
import static org.testng.Assert; imports the Assert class from the org.testng package. And it already exists above.

Reply:
already removed it